dd-java-agent/agent-bootstrap/src/main/java/datadog/trace/bootstrap/Agent.java

```diff
@@ -420,6 +423,20 @@ private static void injectAgentArgsConfig(String agentArgs) {
     }
   }
 
+  @SuppressForbidden
+  public static void patchJPSAccess(Instrumentation inst) {
+    if (Platform.isJavaVersionAtLeast(9)) {
+      // Unclear if supported for J9, may need to revisit
+      try {
+        Class.forName("datadog.trace.util.JPMSJPSAccess")
+            .getMethod("patchModuleAccess")
+            .invoke(inst);
+      } catch (Exception e) {
+        log.warn("Failed to patch module access for jvmstat");
```

Comment:
Please, consider sending this warning to telemetry instead. Ideally with the java version info.

```suggestion
        log.deug(SEND_TELEMETRY, "Failed to patch module access for jvmstat and Java version " + Platform.getRuntimeVersion());
```

We can not use the logger placeholder `{}` because the telemetry forwarder will ignore the expansion.